ServerConnection.py

```diff
@@ -0,0 +1,189 @@
+import socket
+import struct
+import json
+from enum import IntEnum
+jsonDict = dict[str, any] # alias
+
+
+class Action(IntEnum):
+    LOGIN = 1
+    LOGOUT = 2
+    MAP = 3
+    GAME_STATE = 4
+    GAME_ACTIONS = 5
+    TURN = 6
+    CHAT = 100
+    MOVE = 101
+    SHOOT = 102
+
+
+class ServerConnection:
+    serverAddress = "wgforge-srv.wargaming.net"
+    serverPort = 443
+
+
+    def __init__(self):
+        '''
+        Opens a socket to the server.
+        '''
+        self.__Socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
This is not a common way of naming private attributes. You'd better use snake case.